projects/plugins/jetpack/modules/custom-css/custom-css.php

```diff
@@ -1218,8 +1368,9 @@ static function publish_box() {
 									 *
 									 * @param string $url Active theme's stylesheet URL. Default to get_stylesheet_uri().
 									 */
-									apply_filters( 'safecss_theme_stylesheet_url', get_stylesheet_uri() )
-								); ?>
+									apply_filters( 'safecss_theme_stylesheet_url', get_stylesheet_uri() ) // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Seems like this would work well here
```suggestion
									esc_url( apply_filters( 'safecss_theme_stylesheet_url', get_stylesheet_uri() ) )
```